src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(Fruit fruit, int quantity) {
+        Storage.fruitsStorage.put(fruit,quantity);
```

Comment:
I do not see "adding" inside the method body. Better call it **set()** or **put()**